Solutions/Endjin.SpecFlow.Selenium.Sample.Specs.Spa/WebsiteUnderTest/web_test.config.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "web_app_name": "Endjin.SpecFlow.Selenium.Sample.Web",
+  "website_url": "http://localhost:1727/",
+  "use_navigation_map": true,
```

Comment:
No, you still use the navigation map for SPA. You just set the SpaRoutingMarker on the map.